lib/sufia/configuration.rb

```diff
@@ -84,10 +84,14 @@ def audit_user_key
       @audit_user_key ||= 'audituser@example.com'
     end
 
-    # TODO: this is called working_path in curation_concerns
```

Comment:
This is a backwards incompatible change and would require deprecating the existing methods.